### DISCUSSION TITLE
QOS 2 support

### DISCUSSION THREAD

**Discussion by om-parashar** [aws/aws-iot-device-sdk-cpp-v2#685] [General]
Noticed that QOS 2 is not supported yet. Is this something that is actively being worked upon by someone? If yes, do we have any timelines for the same?

**Comment by om-parashar** [comment]
created in wrong repo. moving it to java repo

**Reply by jmklix** [reply]
For anyone else interested the answer is the same for all repos, AWS IoT Core doesn't support QOS 2.  https://github.com/aws/aws-iot-device-sdk-java-v2/discussions/549
